packages/apps/desktop-viewer-test/public/locales/en-PSEUDO/iTwinViewer.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
could you run the new pseudolocalize comand and commit those changes?

Reply:
done